### DISCUSSION TITLE
Adding firepits

### DISCUSSION THREAD

**Discussion by Yekara** [organicmaps/organicmaps#3580] [Ideas]
I'd really like firepits (leisure=firepit) to be added to the maps and being searchable.

**Comment by hb0nd** [comment]
would be good for https://github.com/organicmaps/organicmaps/discussions/3189

**Reply by biodranik** [reply]
You can create an issue and find an appropriate vector icon for it.

**Reply by hb0nd** [reply]
will do when i have time 👍
